ui/apps/platform/src/Containers/PolicyCategories/PolicyCategoriesList.tsx

```diff
@@ -2,24 +2,22 @@ import React from 'react';
 import { SimpleList, SimpleListItem } from '@patternfly/react-core';
 
 type PolicyCategoriesListProps = {
-    // policyCategories: {
-    //     id: string;
-    //     name: string;
-    //     isDefault: boolean;
-    // }[];
-    policyCategories: string[];
+    policyCategories: {
+        id: string;
+        name: string;
+        isDefault: boolean;
+    }[];
```

Comment:
It will be possible to import `PolicyCategory` type as a follow up after #2341 has been merged.